lib/phoenix_html/form.ex

```diff
@@ -372,6 +372,56 @@ defmodule Phoenix.HTML.Form do
     html_escape([form_tag(action, options), fun.(form), raw("</form>")])
   end
 
+  @doc """
+  Generate a new form builder for the given parameter in form **without** an
+  anonymous function.
+
+      <%= f = form_for @changeset, Routes.user_path(@conn, :create), opts %>
+        Name: <%= text_input f, :name %>
+
+        <%= for friend_form = inputs_for(f, :friends) do %>
+          <%= text_input friend_form, :name %>
+        <% end %>
+      </form>
+
+  A shortcut for `inputs_for(changeset, url, [])`.
+  """
+  @spec inputs_for(t, field) :: list(Phoenix.HTML.Form.t())
+  def inputs_for(form, field), do: inputs_for(form, field, [])
```

Comment:
We can remove this definition and simply use default arguments in the next clause. :)

Reply:
We can't  use default arguments in the next clause because `input_for/4`:

```
== Compilation error in file lib/phoenix_html/form.ex ==
** (CompileError) lib/phoenix_html/form.ex:438: def inputs_for/4 defaults conflicts with inputs_for/3
```
I'm not sure what I can do in this case? :thinking:

Reply:
I see. I think you can keep this definition then but you can make the docs very short:

```
@doc """
Same as `inputs_for(form, field, [])`.
"""
```